js/layout-javascript/small-card-code.js

```diff
@@ -262,6 +268,11 @@ DynamicList.prototype.attachObservers = function() {
     })
     .on('click', '.small-card-list-item', function(event) {
       var _that = $(this);
+
+      if ($(event.target).hasClass('small-card-bookmark-holder') || $(event.target).parents('.small-card-bookmark-holder').length) {
```

Comment:
I think in this case `$(this)` is the same as `$(event.target)`, which means you already have the jQuery object cached. Just use `_that.hasClass()` and `_that.parents()`.